### DISCUSSION TITLE
Is it possible to slow down speech output?

### DISCUSSION THREAD

**Discussion by pachacamac** [coqui-ai/TTS#2067] [General Q&A]
I'm wondering if it is possible to configure the speed of the output.
I mean both pauses between words and sentences as well as overall pronunciation speed. I'd like to slow it down as much as possible without sounding unnatural and I'd like to avoid post processing options such as [this](https://en.wikipedia.org/wiki/Audio_time_stretching_and_pitch_scaling) if possible because they don't sound good in my opinion.

I mainly use these combinations in case this is relevant:

```
model_name="tts_models/en/vctk/vits"
vocoder_name=built into model
```

and

```
model_name="tts_models/de/thorsten/tacotron2-DDC"
vocoder_name="vocoder_models/de/thorsten/hifigan_v1"
```

This might be a bit of a newbie question but I'd be grateful to be pushed in the right direction :)

**Comment by p0p4k** [comment]
#1659

**Reply by pachacamac** [reply]
Hey @p0p4k thank you for this hint 👍 how would I play with synthesizer.tts_model.length_scale? I assume it would go into one of those three?

```
  --config_path CONFIG_PATH - Path to model config file.
  --vocoder_config_path VOCODER_CONFIG_PATH - Path to vocoder model config file.
  --encoder_config_path ENCODER_CONFIG_PATH - Path to speaker encoder config file.
```

However I can't find many examples. I tried 

```
{
    "model_args": {
        "length_scale": 2.0
    }
}
```

as a config.json for the `--config_path` argument but to no effect.

Is this something I would need to do and then re-train or can this be applied to a pre-trained model?

Could you maybe provide a quick example on how to use this? I'm a bit lost.

**Comment by desis123** [comment]
Change length_scale parameter in config.json file
